src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import { getArticleID } from "../utils";
+import { Card, Stack, Badge } from "react-bootstrap";
+import { FaRegComments } from "react-icons/fa";
+import { LuVote } from "react-icons/lu";
+import { useParams } from "react-router-dom";
+
+export default function ArticlePage() {
+  const [currentArticle, setCurrentArticle] = useState({});
+  const { article_id } = useParams();
```

Comment:
It would be great if we could have a loading state in here to give the user feedback when we're fetching the data 👍